fastn-xtask/src/new_app.rs

```diff
@@ -0,0 +1,185 @@
+pub async fn new_app(app_name: &str) -> fastn_core::Result<()> {
```

Comment:
This won't be in this crate. After I have created an app only will xtask will be present. Fastn new -t or fastn template etc will create new app, not this crate.